### PR TITLE
TER-56 Init TD sharing tests

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: https://keboola.atlassian.net/browse/TER-56
connection: https://github.com/keboola/connection/pull/4062

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/Mixed/StorageApiSharingTestCase.php [hunk 1]
```diff
@@ -126,6 +126,7 @@ protected function initEmptyBucketForSharingTest($name, $stage, $backend)
                 "$stage.c-$name",
                 [
                     'force' => true,
+                    'async' => true,
```
Comment (zajca) [inline tests/Backend/Mixed/StorageApiSharingTestCase.php:129]: :heart:

File: tests/Backend/MixedTeradata/SharingTest.php [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        $token = $this->_client->verifyToken();
+        self::assertTrue($token['admin']['isOrganizationMember']);
+    }
+
+    /**
+     * @throws ClientException
+     * @throws \Doctrine\DBAL\Exception
+     * @throws \Keboola\StorageApi\Exception
+     */
+    public function testAccessDataInLinkedBucketFromWSViaRO(): void
```
Comment (zajca) [inline tests/Backend/MixedTeradata/SharingTest.php:32]: Nice `WSViaRO` :D

File: tests/Backend/MixedTeradata/SharingTest.php [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    use WorkspaceConnectionTrait;
+
+    /**
+     * @return void
+     */
+    public function testOrganizationAdminInTokenVerify(): void
+    {
+        $token = $this->_client->verifyToken();
+        self::assertTrue($token['admin']['isOrganizationMember']);
+    }
```
Comment (zajca) [inline tests/Backend/MixedTeradata/SharingTest.php:25]: proč je to vlastní test? TO by mělo byt v setupu, ne? Aby sa ti nepustil žádný ten test case.

**Review by zajca — COMMENTED** [review]
Přijde mě to v pohodě zatím sem nezkusil, nechápu pointu toho `testOrganizationAdminInTokenVerify`

File: tests/Backend/MixedTeradata/SharingTest.php (hunk 3 above)
Comment (zajca) [reply tests/Backend/MixedTeradata/SharingTest.php:25]: image

Comment (jirkasemmler) [reply tests/Backend/MixedTeradata/SharingTest.php]: zrusil jsem to ze vsech mixed suit. Dela se to uz v setUpu, jen jsem tam presunul assert

**Review by zajca — APPROVED** [review]

**Pull request merged**